removed logic to add +1 to phone numbers

greggraf opened the pull request:
in expectation that the server will provide. Updated logic in href to remove space in phone numbers

nickoneill commented:
Thanks! This'll make it easy to roll this out when we change the backend.

Test fails on a js error, fix looks like this: https://github.com/5calls/5calls/tree/pr/305

oderby commented on static/js/components/contact.js:
```diff
@@ -24,7 +24,7 @@ module.exports = (c, state, prev, send) => {
           <ul class="call__contact__field-office-list">
             ${c.field_offices.map(office => html`
-              <li><a href="tel:+1${office.phone.replace(/-/g, '')}">${office.phone}</a> ${cityFormat(office,c)}</li>
+              <li><a href="tel:${office.phone.replace(/-| /g, '')}">${office.phone}</a> ${cityFormat(office,c)}</li>
```
Visually, it looks cluttered/clumsy to *display* the phone number with the "+1 " prefix, so I think we should remove it. Perhaps we can strip the prefix for the display only? Something like `office.phone.replace(/^\+\d* /, '')` here and below.

greggraf commented:
@oderby - I updated the tests to verify the formatting of both the href and the display text and changed the code with what I think you were suggesting.
